en/docs/references/about-this-release.md

```diff
@@ -98,6 +98,38 @@ The following new features and enhancements (introduced in WSO2 IS 6.0.0) are in
 
     [Learn more]({{base_path}}/deploy/configure-recaptcha/)
 
+-  **Tenant qualified URLs and tenanted sessions**
+
+    From IS 7.0.0 onwards, tenant qualified URLs will be enabled by default, which consistently qualifies every URL/ endpoint of WSO2 Identity Server with the tenant in a path parameter. This improves flexibility for tenant-wise sharding and branding compared to previous releases.
+
+    Additionally from IS 7.0.0 onwards, tenanted sessions will also be enabled allowing sessions and caches to be created in a tenanted space.
+
+    !!! info
+        It is strongly recommended to use WSO2 Identity Server with these two configs being enabled. However, if you need to disable any of these configs, follow the below steps.
```

Comment:
Cannot say `strongly recommended`. We need to mention this is when coming from a previous version and have a requirement to keep these configs disabled

Reply:
Addressed with 185c51fc863cee75da40bf40b57dec329acf3836.
cc: @divyaamunugama